ballerina/client.bal

```diff
@@ -115,11 +115,20 @@ public isolated client class Client {
 #
 # + datasourceName - The driver class name to be used to get the connection
 # + properties - The database properties, which should be applied when getting the connection
+# + mode - Value for the SSL mode
 public type Options record {|
     string? datasourceName = ();
     map<anydata>? properties = ();
+    SSLMode sslMode = DISABLE;
 |};
 
+# Possible values for the SSL mode.
+public enum SSLMode {
+    DISABLE = "disable",
+    VERIFY_CA = "verify-ca",
+    VERIFY_FULL = "verify-full"
```

Comment:
Do we need to assign values for the enums here?

Reply:
VERIFY_CA and VERIFY_FULL because of the underscore, can handle this internally also

Reply:
I think it's better to handle this internally as we don't need to expose this information to the user.